dix/dixfonts.c

```diff
@@ -1868,7 +1869,7 @@ get_client_resolutions(int *num)
     static struct _FontResolution res;
     ScreenPtr pScreen;
 
-    pScreen = screenInfo.screens[0];
+    pScreen = dixGetFirstScreenPtr();
```

Comment:
We could also rename the var to `firstScreen` for the sake of uniformity (here and in other similar cases). Not a big deal though.